src/main/data/facilities.json

```diff
@@ -3853,6 +3853,16 @@
     "code": "LV",
     "description": "LightVAULT"
   },
+  {
+    "code": "LVF",
+    "contact": {
+      "address": "236 rue de la mairie 73220 Epirre, France",
```

Comment:
```suggestion
      "address": "236 rue de la Mairie, 73220 Epirre, France",
```

Reply:
It's a cut/paste from their entry. Do you want me to ask them?

Reply:
> It's a cut/paste from their entry. Do you want me to ask them?

I would definitely ask them to confirm the name of the town.